.github/workflows/ci.yml

```diff
@@ -21,6 +21,7 @@ jobs:
         os:
           - windows-latest
           - ubuntu-latest
+          - ubuntu-24.04-arm
```

Comment:
i am not sure about this... we don't run on macos either.

Reply:
The intention is to make sure it's tested in linux arm environment